### PR TITLE
⬆️ Update dependencies

### PR DESCRIPTION
**Pull request by AileenCGN**
needs #105

- image-size@0.6.1
- validator@9.1.1
- chai@4.1.2
- cz-conventional-changelog@2.1.0
- mocha@4.0.1
- semantic-release@9.1.0

### CODE REVIEW

**Comment by kirrg001** [comment]
rebase needed

**Comment by AileenCGN** [comment]
Rebased!

**Comment by jbhannah** [comment]
hmmm, I thought I had Greenkeeper enabled on here. Shall I see if I can re-enable it?

**Comment by AileenCGN** [comment]
@jbhannah Greenkeeper is activated. I closed the PR, as there was manual work to do.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,13 @@
     "watch": "mocha --watch $(find test -name '*.test.js')",
     "coverage": "istanbul cover ./node_modules/.bin/_mocha -- $(find test -name '*.test.js')",
     "test-travis": "istanbul cover node_modules/.bin/_mocha --report lcovonly -- $(find test -name '*.test.js')",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release"
+  },
+  "engines": {
+    "node": "^4.5.0 || ^6.9.0 || ^8.8.0"
+  },
+  "engines": {
```
Comment (kirrg001) [inline package.json]: this is twice here

**Comment by jbhannah** [comment]
@AileenCGN Ah, gotcha — that initial PR would need to be merged for Greenkeeper to be enabled. I've had to fix that manually before, where I needed to locally check out an initial PR or one for a particular package to get things passing before it could be merged. After this is merged I can try enabling it again.

File: package.json (hunk 1 above)
Comment (AileenCGN) [reply package.json]: Damn you rebase 🙈

File: package.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "watch": "mocha --watch $(find test -name '*.test.js')",
     "coverage": "istanbul cover ./node_modules/.bin/_mocha -- $(find test -name '*.test.js')",
     "test-travis": "istanbul cover node_modules/.bin/_mocha --report lcovonly -- $(find test -name '*.test.js')",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release"
```
Comment (kirrg001) [inline package.json]: Can you explain this change pls?
Never used semantic release, so i have no idea.

Comment (AileenCGN) [reply package.json]: Me neither 😉 

It's part of the breaking changes that came with version 9.0:

https://github.com/semantic-release/semantic-release/releases/tag/v9.0.0

Comment (kirrg001) [reply package.json]: Alright 😬

**Pull request merged**